mapper/tedge_mapper/Cargo.toml

```diff
@@ -8,11 +8,14 @@ description = "tedge_mapper is the mapper that translates thin-edge.io data mode
 
 [package.metadata.deb]
 depends = "tedge"
-maintainer-scripts = "configuration/debian/tedge_mapper"
+maintainer-scripts = "../../configuration/debian/tedge_mapper"
```

Comment:
Don't change this line. We build the package from the repository root directory in our pipeline. It must be the previous path to apply our control scripts.